sources/platform/actors/running/store.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Actors in store
```

Comment:
I've modified this section, please see (https://github.com/apify/apify-docs/pull/535)